### DISCUSSION TITLE
Страница "Что такое FHIR"

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#90] [Страницы Wiki]
На странице http://fhir-ru.zendoc.me/whatsfhir размещены ссылки на обзорные курсы по FHIR и тематические доклады.
В тематических докладах покрыты такие темы:
- Search API
- Профили и расширения
- Терминологии 
Какие темы еще надо включать? Есть ли по ним материалы?

Предложения по дополнению и изменению странички прошу писать в этой дискуссии.

**Comment by AndreyAlabin** [comment]
Lloyd McKenzie - FHIR Architecture Design | DevDays 2019 Amsterdam
https://www.youtube.com/watch?v=qSSrJaS6aBY
